src/package.json

```diff
@@ -72,13 +74,15 @@
     "express-session": "^1.17.3",
     "helmet": "^6.0.0",
     "knex": "^2.4.2",
+    "monitor": "file:",
```

Comment:
I'm intrigued, tell me more! ❤️ 
I was assuming this would let you do absolute imports like `import * as foo from "monitor/lib/bar.js"`, but not sure I saw anywhere else in this PR that is referencing this "monitor" pseudo package.